### PR TITLE
[CTB-83] Grouping of SQL queries

### PR DESCRIPTION
**Pull request by PiotrDeda**

### CODE REVIEW

File: src/server/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 flask~=2.2.3
-gunicorn~=20.0.4
+gunicorn==20.1.0
 pandas~=1.5.3
-pyopenssl~=23.0.0
+pyopenssl==23.1.0
```
Comment (unlvy) [inline src/server/requirements.txt:4]: Why == and not ~=? I am aware that those two are most recent releases, but keep in mind that new releases will come in the future and we will most probably want to use those

File: src/server/constants.py [hunk 2]
```diff
@@ -13,3 +13,17 @@ class PATHS:
     DATABASE_SCHEMA = RESOURCES + "schema.sql"
     DATABASE_DEFAULT_DUMP = RESOURCES + "dump.sql"
     DATABASE = VAR_PATH + "database_ctb.db"
+
+
+class QUERIES:
```
Comment (unlvy) [inline src/server/constants.py:18]: Please add @dataclass(frozen=True) above `class Queries:` so that fields of this class become immutable

**Review by unlvy — CHANGES_REQUESTED** [review]
In future, please use more descriptive commit messages

File: src/server/requirements.txt (hunk 1 above)
Comment (PiotrDeda) [reply src/server/requirements.txt:4]: PyCharm's doing 😅 I'll fix it

**Review by unlvy — APPROVED** [review]

**Review by vis4rd — APPROVED** [review]

**Pull request merged**